api-tester/src/main/java/com/revenuecat/apitester/java/PurchasesAPI.java

```diff
@@ -196,7 +196,7 @@ static void checkConfiguration(final Context context,
                 .observerMode(false)
                 .service(executorService)
                 .diagnosticsEnabled(true)
-                .entitlementVerificationMode(EntitlementVerificationMode.INFORMATIONAL)
```

Comment:
Right, I thought the other name made it clearer that diagnostics was being enabled... Also, once the feature goes out of beta, I would consider this API deprecated and just split it back into 2 options, so that's why I thought to just making it a boolean since we currently have 2 states only. Thoughts?

Reply:
Oooh. I thought we would always force both things to work together, not just during the beta. In that case this makes sense 👍🏻

Reply:
Hmm not sure, but IMO, these seem like separate features that should be able to be configured separately. What do you think @aboedo?